internal/controller/ciliumflow/flow_cilium.go

```diff
@@ -15,56 +15,56 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-// FlowManager holds a logger and Cilium client
-type FlowManager struct {
+// Collector collects flows from Cilium Hubble Relay running in this cluster.
+type Collector struct {
 	logger *zap.SugaredLogger
 	client observer.ObserverClient
 }
 
 // discoverHubbleRelayAddress uses a kubernetes clientset in order to discover the address of hubble-relay within kube-system.
-func discoverHubbleRelayAddress(ctx context.Context, logger *zap.SugaredLogger, clientset kubernetes.Interface) (string, error) {
-	service, err := clientset.CoreV1().Services("kube-system").Get(ctx, "hubble-relay", metav1.GetOptions{})
+func discoverHubbleRelayAddress(ctx context.Context, logger *zap.SugaredLogger, ciliumNamespace string, clientset kubernetes.Interface) (string, error) {
+	service, err := clientset.CoreV1().Services("kube-system").Get(ctx, ciliumNamespace, metav1.GetOptions{})
```

Comment:
You replaced the wrong literal.
```suggestion
	service, err := clientset.CoreV1().Services(ciliumNamespace).Get(ctx, "hubble-relay", metav1.GetOptions{})
```
You should define `"hubble-relay"` as a constant in this file.